.github/workflows/core-dependency-update.yml

```
@@ -59,4 +59,40 @@ jobs:
           echo "📦 Extracted core version: ${CORE_VERSION}"
           
           # Run the complete core dependency update pipeline
-          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}" 
\ No newline at end of file
+          node run-pipeline.mjs core-dependency-update "${CORE_VERSION}"
+  
+codecov:
```

actionlint (1.7.7)

unexpected key "codecov" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)